sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LogicalPlan.scala

```diff
@@ -95,6 +95,12 @@ abstract class LogicalPlan extends QueryPlan[LogicalPlan] with Logging {
   }
 
   /**
+   * Estimate complete [[Statistics]] for this plan. Each plan need to override this to get
+   * complete statistics.
+   */
+  def completeStats: Statistics = statistics
```

Comment:
What is the difference with the other statistics field? Could you give an example?